src/components/ui/CaptionInput.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+
+import CircleIcon from './CircleIcon'
+import IconButton from './IconButton'
+import Input from './Input'
+import ListEntry from './ListEntry'
+
+const StyledCaptionInput = styled(ListEntry)`
+  padding: 8px;
+  height: 33px;
+  display: flex;
+  justify-content: space-between;
+`
+
+const CaptionInput = ({ value, onChange, onDelete, onClick, image }) => (
+  <StyledCaptionInput
+    leftIcons={<CircleIcon>{image}</CircleIcon>}
+    primaryText={
+      <Input placeholder="Add a caption..." onChange={onChange} value={value} />
+    }
+    rightIcons={
+      <IconButton
+        size="44"
+        raised={false}
+        icon={<> &times; </>}
+        onClick={onClick}
+        label="Delete image caption"
+        pressed={onDelete}
```

Comment:
You can omit pressed here! Try to figure out what pressed means in the context of IconButton.